deploy.sh

```
@@ -1,7 +1,43 @@
+source $HOME/.bash_script_aliases
+
+PROJECT=pennantchase-256
+LOCATION=us-west1
+SERVICE=process-box-score
+LOGS_BUCKET=gs://${PROJECT}_build-logs
+REPO=artifacts
+
+TIMESTAMP="$(date -u +'%Y-%m-%dT%H:%M:%S.%NZ')"
+BUILD_LOG="/tmp/${PROJECT}-build-${TIMESTAMP}.log"
+DEPLOY_LOG="/tmp/${PROJECT}-deploy-${TIMESTAMP}.log"
 cd $(realpath "$(dirname "${BASH_SOURCE[0]}")") &&
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)